lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_git_helper.rb

```diff
@@ -12,11 +12,19 @@ module GitHelper
         # @env PROJECT_NAME The name of the directory containing the project code (especially containing the `build.gradle` file)
         #
         def self.commit_version_bump
-          Fastlane::Helper::GitHelper.commit(
-            message: 'Bump version number',
-            files: File.join(ENV['PROJECT_ROOT_FOLDER'], ENV['PROJECT_NAME'], 'build.gradle'),
-            push: true
-          )
+          if ENV['HAS_VERSION_PROPERTIES'].nil?
```

Comment:
Nice of you to support both modes 👏

Reply:
What about using `properties_file_exists` for that test instead, and getting rid of the need for the `HAS_VERSION_PROPERTIES` env var altogether, relying on `properties_file_exists` to auto-detect the right case?

Reply:
@JavonDavis reiterating my comment here since you don't seem to have addressed this since my last review?